packages/mobile/src/invite/saga.ts

```diff
@@ -85,20 +84,18 @@ export function getInvitationVerificationFeeInWei() {
 export async function generateInviteLink(inviteCode: string) {
   let bundleId = DeviceInfo.getBundleId()
   bundleId = bundleId.replace(/\.(debug|dev)$/g, '.alfajores')
-  const encodedInvite = encodeURIComponent(`invite-code=${inviteCode}`)
 
-  // Android part
-  const playStoreLink = await VersionCheck.getPlayStoreUrl({ packageName: bundleId })
-  const playStoreUrl = `${playStoreLink}&referrer=${encodedInvite}`
-
-  // iOS part
-  const appStoreId = await getAppStoreId(bundleId)
-  const appStoreUrl = await VersionCheck.getAppStoreUrl({ appID: appStoreId })
+  // trying to fetch appStoreId needed to build a dynamic link
+  let appStoreId
+  try {
+    appStoreId = await getAppStoreId(bundleId)
+  } catch (error) {
+    Logger.error(TAG, 'Failed to load AppStore ID: ' + error.toString())
```

Comment:
what happens in this case? Can we still generate a link even without appStoreId?

Reply:
Yes, it would just open `https://celo.org/build/wallet` on iOS instead of AppStore link